docs/examples/plot_text_vectorization.py

```diff
@@ -0,0 +1,84 @@
+"""
+.. _plot_text_vectorization.py:
+
+Text Vectorization Example
+==========================
+
+This example illustrates how dask ml can be used to
+vectorize textual data.
+"""
+import os
+import re
+import tarfile
+from glob import glob
+
+import scipy.sparse
+from sklearn.datasets import get_data_home
+from sklearn.externals.six.moves.urllib.request import urlretrieve
+
+import dask
+import dask.bag as db
+import dask.multiprocessing
+
+from dask_ml.feature_extraction.text import HashingVectorizer
+
+dask.set_options(get=dask.multiprocessing.get)
```

Comment:
We'll want to do this on a test-by-test basis rather than globally, otherwise we'll affect other tests

Reply:
Ah, this is an example.  My mistake.